interface_tester/plugin.py

```diff
@@ -328,8 +328,11 @@ def run(self) -> bool:
                 with tester_context(ctx):
                     test_fn()
             except Exception as e:
+                logger.error(f"Interface tester plugin failed with {e}", exc_info=True)
```

Comment:
`logger.exception(...` might also work nice in here?

Reply:
is equivalent, but indeed, didn't know it existed.